maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/UpdateMojo.java

```diff
@@ -133,7 +141,7 @@ private void update(ModifiedPomXMLEventReader pom, List<Dependency> dependencies
     }
 
     private void updateProperties(ModifiedPomXMLEventReader pom, UpdateChecker checker) throws Exception {
-        PROPERTY: for (Map.Entry<Property, PropertyVersions> entry : getHelper().getVersionPropertiesMap(getProject(), /* TODO */ new Property[0], /* TODO */ null, null, true).entrySet()) {
+        PROPERTY: for (Map.Entry<Property, PropertyVersions> entry : getHelper().getVersionPropertiesMap(VersionsHelper.VersionPropertiesMapRequest.builder().withMavenProject(getProject()).build()).entrySet()) {
```

Comment:
Adapting to random upstream breakage as was done in the original commit. Stepped through this in the debugger to make sure it worked.